src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -32,6 +32,14 @@ object TezosTypes {
                               internalOperationResult: Option[AppliedInternalOperationResult]
                               )
 
+  case class Fees(
```

Comment:
This file contains types that correspond to the data returned by the Tezos node. Please move this type to the file where it is actually being used.